keras_cv/layers/preprocessing/channel_shuffle.py

```diff
@@ -0,0 +1,103 @@
+# Copyright 2022 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import tensorflow as tf 
+from tensorflow.keras import layers, backend
+
+class ChannelShuffle(layers.Layer):
+    """ChannelShuffle performs shuffling the channel of inputs.  
+
+    Input shape:
+        The expected images should be [0-255] pixel ranges.
+        3D (unbatched) or 4D (batched) tensor with shape:
+        `(..., height, width, channels)`, in `"channels_last"` format
+    Output shape:
+        3D (unbatched) or 4D (batched) tensor with shape:
+        `(..., height, width, channels)`, in `"channels_last"` format
+
+    Args:
+        groups:
+            Number of groups to divide the input channels. Default 3. 
```

Comment:
Remove line break, e.g. "groups: Number of..."

Same below.

Reply:
Done.